lib/components/narrative/default/itinerary.css

```diff
@@ -1,9 +1,5 @@
 /* ITINERARY OPTION */
 
-.otp .option.default-itin button:focus {
-  outline: none;
-}
-
```

Comment:
This makes me a bit nervous -- I think this might affect some other things? Can we add a `:not` instead?

Reply:
`:not(button)`? or something else?

Reply:
Addressed in [89b7aa3](https://github.com/opentripplanner/otp-react-redux/pull/815/commits/89b7aa3aaac2bba7d423c34c88ea51e7bbed9416) but i'm worried it has the same function as just deleting it. Let me know what you think!!